js/util/extractProps.ts

```diff
@@ -42,8 +45,21 @@ export const extractProps = (
   }
 
   let classCriteria: PropExtractorCriteria = EMPTY_CRITERIA;
-  if (fiberNode.stateNode && fiberNode.stateNode.heapOptions && fiberNode.stateNode.heapOptions.eventProps) {
-    classCriteria = fiberNode.stateNode.heapOptions.eventProps as PropExtractorCriteria;
+  if (
```

Comment:
I think it would be good to add a comment here around why we are expecting these things to exist, and on which components (e.g. Class based components will have a fiberNode, and functional components will not, etc.)

Reply:
done